src/js/popup.js

```diff
@@ -84,6 +84,20 @@ function showNagMaybe() {
 
   function _showError(error_text) {
     $('#instruction-text').hide();
+    $('#error-message').text(error_text);
+    // if error is the 'unexpected error occurred' Firefox issue commonly associated with low disk space, show special message
+    if (typeof browser == "object" && typeof browser.runtime.getBrowserInfo == "function" && error_text.includes("unexpected")) {
```

Comment:
Let's look for the exact entire string ("An unexpected error occurred" I think) and remove the browser detection. It's unlikely a different browser will have this exact same error string, but if it does happen, there is no big problem and we will eventually learn of it and reassess.

The browser detection here complicates the logic for not enough benefit in my estimate. The browser lookup is async; there is a brief flash of the "let us know" UI before it gets hidden.